scripts/generateContent.gradle

```diff
@@ -0,0 +1,74 @@
+buildscript {
+    repositories {
+        maven {
+            credentials {
+                username mavenUsername
+                password mavenPassword
+            }
+            url mavenRepository
+        }
+    }
+    dependencies {
+        classpath 'com.theokanning.openai-gpt3-java:service:0.12.0'
+    }
+}
+
+import com.theokanning.openai.service.OpenAiService;
+import com.theokanning.openai.completion.CompletionRequest;
+
+//tag::generateContent[]
+task generateContent (
+    description: 'Generate documentation with the help of OpenAI',
+    group: 'docToolchain'
+) {
+    doLast{
+        def token  =  config.openAI.token?:null
```

Comment:
I think it would make sense to provide the means so that configuration parameters can be provided via command line. 
This way the user doesn't have to change the configuration files in case they want to call the task just once.

```
def token = (findProperty("openAI.token")?:config.openAI.token)?:null
```

Reply:
good chatch, thanks!